test-cloud/test-cloud-performance/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iJbpmEJBTimerPerfIntegrationTest.java

```diff
@@ -23,198 +23,33 @@
 import java.time.OffsetDateTime;
 import java.time.temporal.ChronoUnit;
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
-import java.util.function.BooleanSupplier;
-
 import org.apache.commons.csv.CSVFormat;
 import org.apache.commons.csv.CSVPrinter;
-import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameter;
-import org.junit.runners.Parameterized.Parameters;
-import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
-import org.kie.cloud.api.DeploymentScenarioBuilderFactoryLoader;
-import org.kie.cloud.api.deployment.Instance;
-import org.kie.cloud.api.scenario.WorkbenchRuntimeSmartRouterImmutableKieServerWithDatabaseScenario;
-import org.kie.cloud.common.provider.KieServerClientProvider;
-import org.kie.cloud.integrationtests.category.Performance;
-import org.kie.cloud.provider.git.Git;
-import org.kie.cloud.tests.common.AbstractMethodIsolatedCloudIntegrationTest;
-import org.kie.cloud.tests.common.client.util.Kjar;
-import org.kie.cloud.tests.common.client.util.RunnableWrapper;
 import org.kie.cloud.tests.common.time.TimeUtils;
-import org.kie.server.api.exception.KieServicesHttpException;
-import org.kie.server.api.model.KieContainerResource;
-import org.kie.server.api.model.ReleaseId;
-import org.kie.server.api.model.definition.QueryFilterSpec;
-import org.kie.server.api.model.instance.ProcessInstance;
-import org.kie.server.api.util.QueryFilterSpecBuilder;
-import org.kie.server.client.KieServicesClient;
-import org.kie.server.client.ProcessServicesClient;
-import org.kie.server.client.QueryServicesClient;
-import org.kie.server.client.UserTaskServicesClient;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.jbpm.process.instance.ProcessInstance.STATE_ACTIVE;
-import static org.jbpm.process.instance.ProcessInstance.STATE_COMPLETED;
 
 @RunWith(Parameterized.class)
-public class KieServerS2iJbpmEJBTimerPerfIntegrationTest extends AbstractMethodIsolatedCloudIntegrationTest<WorkbenchRuntimeSmartRouterImmutableKieServerWithDatabaseScenario> {
+public class KieServerS2iJbpmEJBTimerPerfIntegrationTest extends BaseJbpmEJBTimersPerfIntegrationTest {
```

Comment:
Just one more thing - the name of this class should be aligned with the other scenario. So this can be something like `JbpmEJBSinglePointTimersPerfIntegrationTest` or something which would reflect the scenario.

Reply:
Done, also updated README with the updated name